run.py

```diff
@@ -582,6 +754,9 @@ def monitor(cheapest=False):
     deregistertask(ECS_TASK_NAME,ecs)
     print("Removing cluster if it's not the default and not otherwise in use")
     removeClusterIfUnused(monitorcluster, ecs)
+    # Remove Cloudwatch dashboard if created and cleanup desired
+    if CREATE_DASHBOARD and CLEAN_DASHBOARD:
```

Comment:
Typically, with the monitor, we want to pull stuff from the monitor file, not the current configuration, because someone might be running two things in DS, and if they start 1, start 2, and then 1 finishes, the config file will reflect 2 not 1. Can we therefore have clean_dashboard take a string (rather than pulling the current app name) and pass it monitorapp instead? Thanks!